uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/sub-controllers/request-modal-details-toil.controller.spec.js

```diff
@@ -343,8 +342,26 @@ define([
             expect(Object.keys(controller.toilAmounts).length).toBeGreaterThan(0);
           });
 
-          it('does not cache TOIL accrual options', function () {
-            expect(OptionGroup.valuesOf.calls.mostRecent().args[2]).toBe(false);
+          it('fetches updated TOIL accrual options from the backend', function () {
+            var retreiveCachedTOILOptions = OptionGroup.valuesOf.calls.mostRecent().args[1];
+
+            expect(retreiveCachedTOILOptions).toBe(false);
+          });
+        });
+      });
+    });
+
+    describe('TOIL accrual options group editor icon', function () {
+      ['admin-dashboard', 'absence-tab'].forEach(function (siteSection) {
+        describe('when the leave request modal is opened in ' + siteSection + ' section', function () {
+          beforeEach(function () {
+            $rootScope.section = siteSection;
+
+            compileComponent({ request: TOILRequestInstance.init() });
+          });
+
+          it('shows the editor icon', function () {
+            expect(controller.showTOILAccrualsOptionEditorIcon).toBe(true);
```

Comment:
We are missing a test for sections that should not show the TOIL accrual icon.

Reply:
Done.